labelbox/schema/project.py

```diff
@@ -617,7 +617,9 @@ def create_batch(self, name: str, data_rows: List[str], priority: int = 5):
             }
         }
 
-        res = self.client.execute(query_str, params,
+        res = self.client.execute(query_str,
+                                  params,
+                                  timeout=180.0,
```

Comment:
Since this is change is only to accommodate `project.create_batch`, can we instead update the timeout for the specific method?

https://github.com/Labelbox/labelbox-python/blob/47063d7f1905f7df7f3696b3ff32ebfc6bc1ae19/labelbox/schema/project.py#L621

`res = self.client.execute(query_str, params,
                                  experimental=True, timeout=180)["project"][method]`

Reply:
Oh, I was misreading the PR. Sorry about that, lgtm!